index.html

```diff
@@ -9,15 +9,84 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
+<body class="padding-app">
```

Comment:
This is not very descriptive, you can write something like `app-container` or someting like that